docs/languages/en/modules/zend.input-filter.intro.rst

```diff
@@ -167,6 +167,197 @@ appropriate object. You may create either ``Input`` or ``InputFilter`` objects i
+                    )
+                );
+            }
+        }
+
+        /**
+         * Filter to ensure an email property is set and > 8 characters and is valid
+         */
+        class EmailInputFilter extends InputFilter
+        {
+            /** @var InputFactory */
+            protected $inputFactory;
+
+            public function __construct()
+            {
+                $this->inputFactory = new InputFactory();
+                $this->setValidators();
+            }
+
+            /**
+             * Loads the validators
+             */
+            protected function setValidators()
+            {
+                $this->setEmailValidator();
+            }
+
+            /**
+             * Creates a validator to check the name property
+             */
+            protected function setEmailValidator()
+            {
+                $this->add(
+                    $this->inputFactory->createInput(
+                        array(
+                            'name'       => 'email',
+                            'required'   => true,
+                            'validators' => array(
+                                array(
+                                    'name' => 'not_empty',
+                                ),
+                                array(
+                                    'name'    => 'string_length',
+                                    'options' => array(
+                                    'min'     => 8
+                                ),
+                                array(
+                                    'name'    => 'email_address',
+                                )
+                            )
+                        )
+                    )
+                );
+            }
+        }
+
+        class SimplePerson implements InputFilterAwareInterface
+        {
+            /** @var string */
+            protected $name;
+
+            /** @var string */
+            protected $email;
+
+            /** @var InputFilter */
+            protected $inputFilter;
+
+            /**
+             * @return string
+             */
+            public function getName()
+            {
+                return $this->name;
+            }
+
+            /**
+             * @param string $name
+             */
+            public function setName($name)
+            {
+                $this->name = $name;
+            }
+
+            /**
+             * @return string
+             */
+            public function getEmail()
+            {
+                return $this->email;
+            }
+
+            /**
+             * @param string $email
+             */
+            public function setEmail($email)
+            {
+                $this->email = $email;
+            }
+
+             /**
+              * Retrieve input filter
+              *
+              * @return InputFilterInterface
+              */
+            public function getInputFilter()
```

Comment:
This method is the actual example: you can actually create a standalone example without the definitions of two input filter classes